dataset/src/main/java/org/keycloak/benchmark/dataset/TaskManager.java

```diff
@@ -81,7 +81,6 @@ public void removeExistingTask(boolean successfullyFinished) {
             Task task = Task.fromMap(existingTask);
             task.SetSuccess(successfullyFinished);
             singleUseObjectProvider.remove(KEY_COMPLETED);
-            singleUseObjectProvider.put(KEY_COMPLETED, TimeUnit.DAYS.toSeconds(1), task.toMap());
```

Comment:
@ahus1 Do we really need this?

This is causing errors in logs when invoking any endpoint. The errors happen if you try to invoke the same endpoint twice. The second attempt will log the error, the third attempt will run fine, the fourth will log the error, etc.

Reply:
The `KEY_COMPLETED` is there to retrieve the status of the previously finished task. There is no other way to retrieve the status of the previous task and to figure out via the API if it was successful or not. 
What kind of error message do you see?